TaskManagement/registration/forms.py

```diff
@@ -0,0 +1,33 @@
+from django import forms
+from allauth.account.forms import SignupForm
+from django.core.validators import RegexValidator
+from phonenumber_field.formfields import PhoneNumberField
+# from .models import User
+
+
+user_role = (
```

Comment:
i guess we have three types of user in our project so why you have taken only one here?